docker/common/docker-run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Let's keep all the scripts in `safety-rules/` for now; we can tackle renaming the image to something like "tcb" later.